modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -1065,6 +1065,127 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
     );
   });
 
+  it('TINY-8639: handling empty text inline elements when root block is empty', () => {
```

Comment:
Do we need any tests for what the default `padd_empty_block_inline_children` value is here (i.e. false)? or is that covered by existing tests?

Reply:
I have added cases for when the option isn't specified. Thanks